File: 15881.py
n = int(input())
s = input()

cnt = 0; i = 0
while i < len(s) - 3:
    if s[i:i+4] == 'pPAp':
        cnt += 1
        i += 4
    else:
        i += 1

print(cnt)

File: 15881.cpp
#include <iostream>
using namespace std;

int main() {
	int n; cin >> n;
	string s; cin >> s;

	int cnt = 0;
	for (int i = 0; i < s.length() - 3; i++) {
		if (s.substr(i, 4) == "pPAp") {
			cnt++;
			i += 3;
		}
	}

	cout << cnt << endl;
}
